index.html

```diff
@@ -2971,27 +2940,30 @@ <h3>Production</h3>
         <h3>Consumption</h3>
 
         <p>
-The <a>DID document</a> MUST be deserialized as a JSON document according to
-the <a href="#consumption">consumption rules for JSON</a>, with one additional
-requirement: The serialized <a>DID document</a> MUST include the
-<code>@context</code> member and be processed according to the rules below.
+All data structures expressed by a JSON-LD <a>representation</a> MUST be
```

Comment:
Same comment as above. Data model and representations apply to DID documents, not other data structures.

Reply:
technically and INFRA map with only an `id` being required could represent anything... and still be a DID Document... this is a side effect  / feature of the abstract data model and supporting unregistered properties.

Reply:
Agree with @OR13 -- we are using a data model that is capable of expressing all the data structures in the specification. A DID Document is one of those data structures. A verification method is another. A service is another. The ADM supports expressing all of those things (because it supports expressing maps). 

To put it another way... the Java language gives you the Object class... it doesn't then say, "You can only use the Object class to express DID Documents." <-- that would be an unnecessary restriction, which would mean that you'd need to create yet another thing to express all the other data structures you need to use.

As an aside, this is why creating bespoke abstract data models for specifications is such a bad idea -- you get pulled down into these esoteric, academic tarpits... but since we were all pulled into this tarpit, we do have a fairly generalized data model (thanks to INFRA) and it can be applied to all our data structures (because other people went through the pain of defining INFRA so that this was possible).

Reply:
@peacekeeper I'll put the language back to what I believe you're comfortable with in https://github.com/w3c/did-core/pull/654/files#r577651396 -- this means that the only valid data structures are the DID Document and it's data structures. As a result, no change is needed for this text.

Reply:
@peacekeeper applied your suggestion in https://github.com/w3c/did-core/pull/654/files#r577928397 to address your objection.